automated/linux/jobs-definition/job-0001_test-repo.yaml

```diff
@@ -18,7 +18,7 @@ actions:
 - boot:
     method: ssh
     connection: ssh
-    prompts: ["root@RevPi32961:"]
+    prompts: ["root@RevPi"]
```

Comment:
before factory reset, the prompt contains even no "RevPi"

Reply:
Just only root@ ?
Ok... I didn't know that...
@ferdrich-kunbus What do you think?

Reply:
I found this note in the documentation

> Note
prompts - it is essential that the test writer records the information on what prompt will be presented when the system boots into that image successfully. If LAVA fails to match device output to provided prompt string the job will fail with timeout on auto_login_action.

You can read it here: [https://validation.linaro.org/static/docs/v2/explain_first_job.html#boot-action-for-qemu](url)

In that case it has to be changed to "root@"...